packages/core/src/domain/configuration/experimentalFeatures.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  updateExperimentalFeatures,
+  isExperimentalFeatureEnabled,
+  resetExperimentalFeatures,
+} from './experimentalFeatures'
+
+describe('experimentalFeatures', () => {
+  it('initial state is empty', () => {
+    resetExperimentalFeatures()
+    expect(isExperimentalFeatureEnabled(undefined as any)).toBeFalse()
+    expect(isExperimentalFeatureEnabled('foo')).toBeFalse()
+    expect(isExperimentalFeatureEnabled('bar')).toBeFalse()
+    expect(isExperimentalFeatureEnabled('')).toBeFalse()
```

Comment:
do we want to use this API with `''`?

Reply:
Just checking as a valid common string. I haven't seen a spec around which string types we care about (uppercase, "Turkish dotless i", diacritics, emoji). 

Do we have a team preference on the extent of string validation?